models/git/lfs.go

```diff
@@ -144,16 +144,17 @@ func NewLFSMetaObject(ctx context.Context, m *LFSMetaObject) (*LFSMetaObject, er
 	}
 	defer committer.Close()
 
-	has, err := db.GetByBean(ctx, m)
-	if err != nil {
+	m, err := db.Get[LFSMetaObject](ctx, builder.Eq{"repo_id": repoID, "oid": p.Oid})
+	if err != nil && !db.IsErrNotExist(err) {
 		return nil, err
 	}
 
-	if has {
+	if err == nil {
 		m.Existing = true
 		return m, committer.Commit()
 	}
```

Comment:
Something is strange here.
That function only `gets` so far. why commit in this case?

Reply:
This will follow previous behaviour. I think we can improve it later.